r/R/feather.R

```diff
@@ -43,60 +87,15 @@ write_feather <- function(x, sink) {
     on.exit(sink$close())
   }
   assert_is(sink, "OutputStream")
-
-  writer <- FeatherTableWriter$create(sink)
-  ipc___TableWriter__RecordBatch__WriteFeather(writer, x)
+  ipc___WriteFeather__RecordBatch(sink, x, version, chunk_size, compression, compression_level)
```

Comment:
@wesm It's odd to me that the writer seems only to write RecordBatches, but the reader returns a Table. Is that just a shortcoming of the bindings?

Reply:
It's a shortcoming yeah. It would be good to change it to work with either RecordBatch or Table

Reply:
I'm changing things to work with either Table or RecordBatch.